index.html

```diff
@@ -1642,6 +1642,18 @@ <h3>Selective Disclosure</h3>
 all of the credential's claims to be revealed.
 
           </li>
+          <li>
+<b> Hashed Values</b> - With this method, the <a>issuer</a> issue a single
+<a>verifiable credential</a> containing all the subject's <a>claims</a>.
```

Comment:
`all the subject's <a>claims</a>.`
-> `all the <a>issuer</a>'s <a>claims</a> about the <a>subject</a>.`

Reply:
Yes fine